examples/huggingface/pytorch/text-generation/quantization/run_generation.py

```diff
@@ -76,7 +76,9 @@
     type=int,
     help="Calibration dataset max or padding max length.",
 )
-parser.add_argument("--recipes", default=None, help="Recipes for smoothquant.")
+parser.add_argument(

```

Comment:
a bit too complex? we already have alpha, recipe; what else is needed？

Reply:
for smoothquant alpha tune, it should be pass recipes like {'smooth_quant': True, 'smooth_quant_args': {'alpha': 'auto', 'folding': False, 'default_alpha': 0.8, 'auto_alpha_args': {'alpha_min': 0.75, 'alpha_max': 0.99, 'alpha_step': 0.01, 'shared_criterion': 'max'}}}